src/test/java/org/sagebionetworks/web/unitclient/widget/entity/controller/EntityActionControllerImplTest.java

```diff
@@ -2833,54 +2833,6 @@ public void testCreateDeletePlaceDocker() {
     assertEquals(expected, result);
   }
 
-  @Test
-  public void testOnShareNoChange() {
```

Comment:
It looks like we don't have existing tests for the `EntityAccessControlListModalWidget`. Could you update these tests to verify we use this widget?